resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -127,6 +121,11 @@ private[spark] class Client(
             .pods()
             .withLabels(driverKubernetesSelectors)
             .watch(podWatcher)) { _ =>
+          val probePingHttpGet = new HTTPGetActionBuilder()
+            .withScheme(if (driverSubmitSslOptions.enabled) "HTTPS" else "HTTP")
```

Comment:
Uppercasing is done [within the library](https://github.com/dhirajsb/fabric8/commit/7aeec7ecff0ed057ca657bfcfe741abd40528e38) anyway.